loader/custom-metrics.js

```diff
@@ -0,0 +1,44 @@
+var handle = require('handle')
+var loader = require('loader')
+
+var SUPPORTABILITY_METRIC = 'sm'
+var CUSTOM_METRIC = 'cm'
+
+/**
+ * Increments the count of a named metric by 1
+ * @param {string} name - Name of the metric
+ * @param {boolean} [isSupportability] - determines which bucket ('cm' or 'sm') to store the metric
+ * @returns void
+ */
+function incrementCounter (name, isSupportability) {
+  handle('api-storeCustomMetric', [
+    loader.now(),
```

Comment:
I think passing the timestamp through may not be necessary, but I understand that it's being consistent what's already there.